ports/stm32/boards/ARDUINO_PORTENTA_H7/mpconfigboard.h

```diff
@@ -0,0 +1,326 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2022 Arduino SA
+ */
+
+#define MICROPY_HW_BOARD_NAME       "PORTENTA"
+#define MICROPY_HW_MCU_NAME         "STM32H747"
+#define MICROPY_PY_SYS_PLATFORM     "Portenta"
+#define MICROPY_HW_FLASH_FS_LABEL	"portenta"
+#define MICROPY_PY_SYS_STDIO_BUFFER (0)
```

Comment:
Why is this disabled?

Reply:
For no good reason, removed.

Reply:
OK, good.

It's best not to have custom options like this (where possible...) so that the functionality available on boards is more or less uniform across boards and hopefully ports.  That's why `MICROPY_CONFIG_ROM_LEVEL` was introduced, to try and unify config options.